<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FOCUS-QA-10" name="com.focus.testcases.InventoryReportsTest" tests="100" failures="12" timestamp="8 May 2022 11:57:24 GMT" time="1826.724" errors="0">
  <testcase name="checkStockAnalysisByBatchReportWithStockTypeGoodBatches" time="2.896" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkSlowMovingItemsReport" time="12.788" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkSlowMovingItemsReport(InventoryReportsTest.java:670)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkSlowMovingItemsReport -->
  <testcase name="checkAgeingAnalysisByRMAReportBackTrackOption" time="26.095" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkActualConsumptionReportWithFilterOption" time="34.305" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkPeakAndLowBalancesReport" time="11.275" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkMultiLevelStockMovementReportFilterOption" time="33.353" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockAnalysisByBinReportWithFilterOption" time="34.142" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockValuationReportBackTrackOption" time="17.230" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockLedgerReportBackTrackOption" time="7.893" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkBestSellingItemReportBackTrackOption" time="15.816" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkTransactionTypeWiseStockReport" time="24.217" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockLedgerReport" time="12.964" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockReportbyTagReportsWithUserForItemRestriction" time="15.521" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkStockReportbyTagReportsWithUserForItemRestriction(InventoryReportsTest.java:863)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockReportbyTagReportsWithUserForItemRestriction -->
  <testcase name="checkStockAnalysisByBatchReportPrintOption" time="5.330" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkAgeingByItemByBinReportWithWarehouseAndIgnoreInternalTransferForAgeingOptions" time="5.938" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkVirtualStockAnalysisReports" time="11.835" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkMultiLevelStockMovementReportBackTrackOption" time="17.019" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockBalanceByWarehouseReportIncludeServiceTypeItemOption" time="3.941" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkVirtualStockAnalysisReportWithIncludeServiceTypeItemOption" time="3.752" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkMultiLevelStockMovementReports" time="12.584" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkBestSellingItemReportWithSortingOnAndIncludeAllVouchersChkBox" time="3.725" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkLoginWithUserForItemRestriction" time="11.716" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockByItemByBinReportWithFilterOption" time="34.333" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkOpeningStockRegisterReport" time="12.523" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockReservationRegisterReport" time="12.068" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkStockReservationRegisterReport(InventoryReportsTest.java:782)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockReservationRegisterReport -->
  <testcase name="checkStockDetailsByBinsReportFilterOption" time="33.217" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkFastMovingItemsReport" time="12.593" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkFastMovingItemsReport(InventoryReportsTest.java:644)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkFastMovingItemsReport -->
  <testcase name="checkAgeingAnalysisByRMAReportWithFilterOption" time="34.174" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkAgeingAnalysisByRMAReport" time="163.103" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkAgeingByItemByBinReportBackTrackOption" time="27.488" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockStatementReportPrintOption" time="5.346" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockAnalysisByBatchReportFilterOption" time="34.074" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockLedgerReportWithMovedItemsOnlyAndIncludeServiceTypeItemsCheckBoxs" time="6.473" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockReportbyTagReportPrintOption" time="5.222" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockReportbyTagReports" time="12.717" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkStockReportbyTagReports(InventoryReportsTest.java:242)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockReportbyTagReports -->
  <testcase name="checkExpiredStockByBinsReports" time="16.406" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkAgeingAnalysisByBatchReportWithWarehouseAndIgnoreInternalTransferForAgeingOptions" time="8.374" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockReservationRegisterReportBackTrackOption" time="7.752" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkActualConsumptionReportBackTrackOption" time="21.105" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkABCAnalysisReportWithClassificationAndCriteriaOptions" time="3.442" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockAgeingAnalysisReports" time="18.615" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockAgeingAnalysisReportWithBackTrackOptions" time="18.213" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkAgeingAnalysisByRMAReportWithWarehouseAndIgnoreInternalTransferForAgeingOptions" time="4.612" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkABCAnalysisReports" time="10.821" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkABCAnalysisReports(InventoryReportsTest.java:269)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkABCAnalysisReports -->
  <testcase name="checkStockDetailsByBinsReportBackTrackOption" time="17.953" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockBalancesByItemByBinsReportPrintOption" time="5.557" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkBestSellingItemReport" time="10.168" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkMultiLevelStockMovementReportWithIncludeServiceTypeItemOption" time="4.559" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkPeakAndLowBalancesReportWithFilterOption" time="32.396" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkPeakAndLowBalancesReportWithFilterOption(InventoryReportsTest.java:705)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkPeakAndLowBalancesReportWithFilterOption -->
  <testcase name="checkStockStatementReportWithIncludeServiceTypeItem" time="6.112" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkStockStatementReportWithIncludeServiceTypeItem(InventoryReportsTest.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockStatementReportWithIncludeServiceTypeItem -->
  <testcase name="checkTransactionTypeWiseStockReportBackTrackOption" time="20.937" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkExpiredStockByBinsReportCustomizationOption" time="14.350" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockTransferRegisterReportPrintOption" time="5.485" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkTransactionTypeWiseStockReportWithFilterOption" time="39.725" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockMovementReportsWithStockMovementByWarehouseAndIncludeServiceTypeItemOptions" time="7.006" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkStockMovementReportsWithStockMovementByWarehouseAndIncludeServiceTypeItemOptions(InventoryReportsTest.java:125)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockMovementReportsWithStockMovementByWarehouseAndIncludeServiceTypeItemOptions -->
  <testcase name="checkStockStatementReport" time="13.583" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkStockStatementReport(InventoryReportsTest.java:91)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkStockStatementReport -->
  <testcase name="checkStockAgeingAnalysisReportWithOptions" time="7.667" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockBalanceByWarehouseReportBackTrackOption" time="18.648" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockLedgerCustomizationReport" time="29.137" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockAnalysisByRmaReports" time="22.230" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockTransferRegisterReportWithFromAndToWarehouseAndDetailsOptions" time="10.337" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockBalancesByItemByBinsReports" time="16.918" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockMovementReportBackTrackOption" time="18.527" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkActualConsumptionReport" time="15.886" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkLoginWithSU" time="11.305" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkPeakAndLowBalancesReportBackTrackOption" time="17.993" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockAnalysisByBatchReportBackTrackOption" time="17.861" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockTransferRegisterReport" time="14.263" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkOpeningStockRegisterReportBackTrackOption" time="7.880" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockBalanaceByBinsReportBackTrackOption" time="18.076" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockAnalysisByBatchReport" time="10.551" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkAgeingAnalysisByBinReportBackTrackOption" time="24.260" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkSlowMovingItemsReportWithFilterOption" time="32.659" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkFastMovingItemsReportWithFilterOption" time="36.445" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkAgeingAnalysisByBinReportWithWarehouseAndIgnoreInternalTransferForAgeingOptions" time="5.876" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkReorderReports" time="9.905" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockTransferRegisterReportBackTrackOption" time="20.530" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockDetailsByBinsReports" time="15.906" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkReorderReportPrintOption" time="0.647" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockAnalysisByRmaReportsPrintOption" time="5.200" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkAgeingStockByBinsReports" time="19.249" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkAgeingStockByBinsReports(InventoryReportsTest.java:561)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkAgeingStockByBinsReports -->
  <testcase name="checkFastMovingItemsReportBackTrackOption" time="18.337" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkAgeingAnalysisByBatchReportBackTrackOption" time="20.377" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockBalanaceByBinsReportFilterOption" time="31.310" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockBalanaceByBinsReports" time="16.776" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockReservationRegisterReportWithFilterOption" time="36.547" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockValuationReports" time="12.584" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkAgeingByItemByBinsReports" time="18.338" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockMovementReports" time="12.637" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockValuationCustomizationReport" time="36.362" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockBalanceByWarehouseReports" time="12.720" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockReportbyTagReportBackTrackOption" time="18.104" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockValuationReportWithStockValuationDisplayLinearReportIncludeServiceTypeItemOption" time="3.977" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkItemQueryReport" time="20.129" classname="com.focus.testcases.InventoryReportsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.InventoryReportsTest.checkItemQueryReport(InventoryReportsTest.java:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkItemQueryReport -->
  <testcase name="checkAgeingAnalysisByBatchReports" time="18.192" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkVirtualStockAnalysisReportBackTrackOption" time="16.953" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkSlowMovingItemsReportBackTrackOption" time="18.054" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockMovementReportWithFilterOption" time="31.512" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkStockAgeingAnalysisReportFilterOption" time="35.827" classname="com.focus.testcases.InventoryReportsTest"/>
  <testcase name="checkOpeningStockRegisterReportFilterOption" time="35.195" classname="com.focus.testcases.InventoryReportsTest"/>
</testsuite> <!-- com.focus.testcases.InventoryReportsTest -->
